docs/README.md

```diff
@@ -0,0 +1,139 @@
+# Piri Documentation
+
+Welcome to the Piri documentation!
+Piri is the software run by all storage providers on the Storacha network. 
+Piri can run entirely on its own with no software other than Filecoin Lotus, or it can integrate into Filecoin storage provider operation running Curio.
+
+## 🚀 Quick Start
+
+New to Piri? Start here:
+
+1. **[Getting Started Guide](./getting-started.md)** - Decision tree to help you choose the right setup
+2. **[Architecture Overview](./architecture.md)** - Understand how Piri components work together
+3. **[Prerequisites](./common/prerequisites.md)** - Ensure your system is ready
+
+## 📚 Documentation Structure
+
+### Common Procedures
+
+Reusable guides referenced throughout the documentation:
+
+- **[Prerequisites](./common/prerequisites.md)** - System requirements and dependencies
+- **[Piri Installation](./common/piri-installation.md)** - Installing the Piri binary
+- **[PEM File Generation](./common/pem-file-generation.md)** - Creating and managing Ed25519 keys
+- **[TLS Termination](./common/tls-termination.md)** - Setting up HTTPS with Nginx or alternatives
+
+### Setup Guides
+
+Step-by-step guides for different components:
+
+- **[PDP Server Setup](./guides/pdp-server-piri.md)** - Run a Piri PDP server for data storage and proofs
+- **[UCAN Server Setup](./guides/ucan-server.md)** - Run a Piri UCAN server for client uploads
```

Comment:
should these be linked here? After all, nobody will need these guides from the beginning. If they are new providers and want to run the full thing they'll use the full stack guide, which can then point to the respective, in-detail guides for the PDP and UCAN servers. And if they are existing providers wanting to re-use their Curio node, they'll use the integration guide, that links to the UCAN server guide.